backend/src/main/backend/Table.java

```diff
@@ -0,0 +1,100 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class Table {
+    private String name;
+
+    private Connection mConnection;
+    private PreparedStatement mCreateTable;
+    private PreparedStatement mDropTable;
+    private PreparedStatement mDeleteRow;
+    private PreparedStatement mInsertRow;
+    private PreparedStatement mUpdateRow;
+    // Other prepared statements...
+
+    public Table(String tName){
+        name = tName;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public void create(Database db) {
+        try {
+            if(name.equals("tblMessages")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblMessages (row_id SERIAL PRIMARY KEY, subject VARCHAR(50) NOT NULL, content VARCHAR(500) NOT NULL, user_id INTEGER REFERENCES tblUsers)");
+            }
+            else if(name.equals("tblComment")) {
+                db.mCreateTable =  db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblComment (row_id SERIAL PRIMARY KEY, message_id INTEGER REFERENCES tblMessages, user_id INTEGER REFERENCES tblUsers, content VARCHAR(500) NOT NULL)");
+            }
+            else if(name.equals("tblUsers")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblUsers (row_id SERIAL PRIMARY KEY, user_name VARCHAR(500) NOT NULL, first_name VARCHAR(500) NOT NULL, last_name VARCHAR(500) NOT NULL, email VARCHAR(500) NOT NULL)");
+            }
+            else if(name.equals("tblLikes")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblLikes (row_id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES tblUsers, message_id INTEGER REFERENCES tblMessages, content INTEGER)");    
+            }
+
+            db.mCreateTable.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
In the app lifecycle, what do we want the user/developer experience to be?
If we fail to create the tables necessary for the backend to run, do we want to allow the app to continue startup? Or should it terminate?
If the application starts up without these tables, what should happen on subsequent CRUD actions?

Reply:
I want the program to terminate. These tables are all necessary for the app to run properly so I don't want the app to just continue startup

Are these questions in response to my catch block? I'm assuming that instead of just printing the stack trace, you want me to perform the desired functionality for when the process fails?